Tests/CMakeOnly/GNUInstallDirs2Other/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 3.2.3)
+
+project(GNUInstallDirsTest)
```

Comment:
Ok, there is a workaround: predefine `CMAKE_SIZEOF_VOID_P` -- it'll be ignored anyway, cuz otherwise (asserting `LIBDIR`) would make tests "unpredictable".